pkg/network/ovn_kubernetes.go

```diff
@@ -877,12 +906,12 @@ func validateOVNKubernetes(conf *operv1.NetworkSpec) []error {
 
 	return out
 }
-
 func getOVNEncapOverhead(conf *operv1.NetworkSpec) uint32 {
 	const geneveOverhead = 100
 	const ipsecOverhead = 46 // Transport mode, AES-GCM
 	var encapOverhead uint32 = geneveOverhead
-	if conf.DefaultNetwork.OVNKubernetesConfig.IPsecConfig != nil {
+	mode := getIPsecMode(conf.DefaultNetwork.OVNKubernetesConfig)
+	if mode == operv1.IPsecModeFull || mode == "" {
```

Comment:
`mode == ""` is not possible?

Reply:
`mode == ""` is possible during upgrade (for bw compatibility, when ipsec was enabled with `ipsecConfig: {}` , but the user can't set `mode` to "" anymore

Reply:
indeed IIUC `getIPsecMode` can't return ""

Reply:
fixed